### PR TITLE
Backbutton component

### LINKED ISSUE #18: Backbutton component
Create back button for header component

### PR DESCRIPTION
**Pull request by MagdaBer**
- import back button SVG
- write Backbutton function 

- Closes #18

### CODE REVIEW

File: src/app/components/Backbutton/Backbutton.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+type BackbuttonProps = {
+  className?: string;
+};
+
+function Backbutton({ className }: BackbuttonProps): JSX.Element {
+  return (
+    <button className={`${styles.button} ${className}`}>
+      <BackIcon width="0.9rem" height="1.5rem" />
+    </button>
```
Comment (marwinburesch) [inline src/app/components/Backbutton/Backbutton.tsx:13]: Also needs an `onClick` prop

**Review by marwinburesch — COMMENTED** [review]
👍

File: src/app/components/Backbutton/Backbutton.tsx (hunk 1 above)
Comment (MagdaBer) [reply src/app/components/Backbutton/Backbutton.tsx:13]: Could you please check if this is the correct solution

**Review by marwinburesch — COMMENTED** [review]
👍

**Pull request merged**